pkg/database/bigquery.schema.json

```diff
@@ -82,5 +82,22 @@
     "mode": "REPEATED",
     "name": "nodes",
     "type": "RECORD"
+  },
+  {
+    "fields": [
+      {
+        "mode": "REQUIRED",
```

Comment:
We can't REALLY add new required fields - bq doesn't allow it.  it's probably OK to keep it as required HERE, but we should enforce that it is non-empty in code. When I live-add it to the schema in-place, it won't be required.